redux/actions/contactUs.js

```diff
@@ -0,0 +1,20 @@
+import 'common/initialiseFirebase.js'
+import firebase from 'firebase'
+import moment from 'moment'
+
+export const handleNameChange = (inputValue) => {
+  return {type: 'contactUs/HANDLE_NAME_CHANGE', payload: inputValue}
+}
+
+export const handleEmailChange = (inputValue) => {
+  return {type: 'contactUs/HANDLE_EMAIL_CHANGE', payload: inputValue}
+}
+
+export const handleMessageChange = (inputValue) => {
+  return {type: 'contactUs/HANDLE_MESSAGE_CHANGE', payload: inputValue}
+}
+export const onSendMessageClick = (messageData) => {
+  const timeStamp = moment().format('YYMMDDHHmmss')
+  firebase.database().ref(`/projectQueries`).push({...messageData, timeStamp: [timeStamp]})
```

Comment:
not sure why there is a `[]` around timestamp? Can we just use `.format()` as well